### PR TITLE
feat: updated task/milestone/bounty creation modal

### PR DESCRIPTION
**Pull request by juniusfree**
task: https://app.wonderverse.xyz/dashboard?task=57062384688169032&view=grid

The main file for this PR is https://github.com/wondrous-dev/wondrous-frontend/pull/579/files#diff-6505ecf535485316988be1cf9f1fa67a616f74956423359af60fb6566a07e365 

**Demo for the latest updates**
https://www.loom.com/share/c1e024d089cc4e39bc559c503f806665

**Demos**
task: https://www.loom.com/share/4f201d728ad54d0eac9400173c9fa1fa
milestone: https://www.loom.com/share/72897e3b103d49b5b4269acbe1bdcc12
bounty: https://www.loom.com/share/d4437af6cc82404298f2f5f29090bec6

### CODE REVIEW

**Comment by andywong418** [comment]
@juniusfree couple of comments from design: Can we get rid of the scrollbar and also make sure that DAO and pods field icons are not missing? Thanks!

File: wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,1224 @@
...
+  const { entityType, handleClose, resetEntityType, open, parentTaskId } = props;
+  const user = useMe();
+  const [recurrenceType, setRecurrenceType] = useState(null);
+  const [recurrenceValue, setRecurrenceValue] = useState(null);
+  const [fileUploadLoading, setFileUploadLoading] = useState(false);
+  const orgBoard = useOrgBoard();
+  const podBoard = usePodBoard();
+  const userBoard = useUserBoard();
+  const board = orgBoard || podBoard || userBoard;
+  const textLimit = 900;
```
Comment (andywong418) [inline wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx:574]: Let's put this as a const with uppercase `TEXT_LIMIT` on top of file

File: wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,1224 @@
...
+                        position="end"
+                        onClick={() => {
+                          form.setFieldValue('points', null);
+                        }}
+                      >
+                        <CreateEntityAutocompletePopperRenderInputIcon />
+                      </CreateEntityAutocompletePopperRenderInputAdornment>
+                    ),
+                  }}
+                  error={form.errors.points}
```
Comment (andywong418) [inline wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx:1042]: In what scenario would there be an error for points? Just wondering

**Review by andywong418 — COMMENTED** [review]
Just some comments, otherwise this looks good!

File: wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx (hunk 2 above)
Comment (juniusfree) [reply wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx:1089]: - not a number
- not a whole number (if understand points, as in story points in scrum)
- negative number or zero

File: wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,1224 @@
...
+  const filteredEligibleReviewers = eligibleReviewers.filter(
+    (reviewer) => !form.values.reviewerIds.includes(reviewer.id)
+  );
+  return (
+    <CreateEntityForm onSubmit={form.handleSubmit}>
+      <CreateEntityHeader>
+        <CreateEntityHeaderWrapper>
+          <CreateEntityDropdown
+            name="orgId"
+            options={filteredDaoOptions}
```
[the remark this answers is not in the document]
Comment (juniusfree) [reply wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx:678]: This is updated now with a default image if the dao or pod has no profile picture.

**Review by andywong418 — APPROVED** [review]

**Pull request merged**